adapters/eth_format.go

```diff
@@ -66,16 +46,12 @@ func (*EthInt256) Perform(input models.RunResult, _ *store.Store) models.RunResu
 		return input.WithError(err)
 	}
 
-	if err = validateSignedRange(i); err != nil {
-		return input.WithError(err)
-	}
-
-	sh, err := utils.EVMSignedHexNumber(i)
+	sh, err := utils.EVMWordSignedBigInt(i)
 	if err != nil {
 		return input.WithError(err)
 	}
 
-	return input.WithValue(sh)
+	return input.WithValue(common.ToHex(sh))
```

Comment:
This has been marked deprecated by `go-ethereum`:

https://github.com/ethereum/go-ethereum/blob/master/common/bytes.go#L25-L26

With that said, I definitely prefer `common.ToHex` to `hexutil.Encode`. But we do use `hexutil.Decode` already, so it makes sense to bring in `Encode`.

Reply:
Ah, ok, will update 👍